pyirf/interpolation.py

```diff
@@ -0,0 +1,270 @@
+"""
+Functions for performing interpolation of IRF to the values read from the data
+"""
+import numpy as np
+import pandas as pd
+import astropy.units as u
+from astropy.io import fits
+from scipy.interpolate import griddata
+
+# definition from lstchain.io.io copied here to avoid dependency
+dl2_params_lstcam_key = 'dl2/event/telescope/parameters/LST_LSTCam'
+
+
+def read_mean_pars_data(data_file, pars):
```

Comment:
What are we parsing here?

Reply:
pars is for parameters
do you want the name of the variable to be changed?

Reply:
Please don't use unnecessary abbreviations.

Reply:
changed to "parameters"